custom_components/alfen_wallbox/sensor.py

```diff
@@ -149,7 +149,7 @@ class AlfenSensorDescription(SensorEntityDescription, AlfenSensorDescriptionMixi
 ALLOWED_PHASE_DICT: Final[dict[int, str]] = {1: "1 Phase", 3: "3 Phases"}
 
 POWER_STATES_DICT: Final[dict[int, str]] = {
-    0: "Unknown",
+    0: "Operative",
```

Comment:
Not sure but should this be 'power main off, bypass off'

Reply:
This is a strange one. I connected my car, 0A charging current. This is the state now:

![IMG_5308](https://github.com/user-attachments/assets/1c0eedcc-246d-45c9-9467-5c7be15d2481)
![IMG_5309](https://github.com/user-attachments/assets/198545e6-2447-461a-a42b-d5bf3bf2b64c)

Reply:
In English: Normal Operation (according to the app)